### DISCUSSION TITLE
Testing new GitHub Beta discussion!

### DISCUSSION THREAD

**Discussion by maoo** [finos/open-developer-platform#103] [General]
@mcleo-d 
@becca
@maoo 

What do you think about this feature?

**Comment by becca** [comment]
Best new feature 🚀 

Excited to iterate with you all on Discussions!

**Comment by maoo** [comment]
I've looked a bit around; so far so good, the feature is quite simple and delivers a publicly accessible URL to point at discussions, which is what we're after.

I find useful to have the `Lock conversation` feature available, it could come handy to mark a discussion "closed".

<img width="454" alt="Screenshot 2020-03-31 at 01 18 39" src="https://user-images.githubusercontent.com/327285/77970890-a5a39c00-72ed-11ea-9889-45e219ef6485.png">

I've also seen the email notification coming by, with the possibility to reply via email, which is a crucial requirement for us.

@becca - few questions:
1. is this feature expected to be available only on repos, and not at org level?
2. The team discussions will stay as they are, or there will be integration points across the 2 features?
3. Any plan to integrate with Github projects?
4. Any plan to integrate with labels?
5. Is there any feature roadmap you can share with us re. upcoming features?

Thank you!

**Reply by becca** [reply]
👋 great questions!
1. available at the repo level only right now
2. would be open to hearing your thoughts on the needs here as you start using Discussions more
3. no plans for this right now. we've been working under the assumption that only work should be represented on a project board so i'd be curious to hear more about if and when you come up against a scenario where you want to track a Discussion on a project board 😄 
4. we're collecting feedback on this now so we can chat more about your needs here too!
5. not at the moment. we're iterating with community feedback so keep me posted on whether there's concerns regarding needs that are not being met in order to use Discussions for the broader Finos community.

**Comment by maoo** [comment]
Thanks @becca .

2. Would be great to be able to create public discussions from the team view as well (see screenshot below)
3. I was thinking of using discussions as notes on project boards; we don't use them much right now, so it's definitely not a priority
4. Definitely +1 on this from our side; it would probably make sense to keep the label dictionary on issues and just "borrow" it :smile:
5. Great. Other than labels, I'd +1 on API support; I'll send you more as soon as we try the feature.

<img width="799" alt="Screenshot 2020-03-31 at 01 40 28" src="https://user-images.githubusercontent.com/327285/77972585-25336a00-72f2-11ea-9fa1-800e0e814a38.png">

**Reply by maoo** [reply]
And I just learned that:
1. conversations are threaded.
2. there's an `answered`/`unanswered` state of the discussion.

@becca - I suppose that only users with Write access can mark/unmark comments as answers, is that correct?

**Reply by becca** [reply]
yes we have threads! 

the folks who can mark/unmark an answer are the admins/maintainers and original author of the post. everyone else has access to `suggest as answer`. we're working on making this workflow more prominent and clearer

**Comment by maoo** [comment]
@becca - yesterday 30 March at 19.30 EST, I've responded to an email notification on this discussion (see attachment), but I don't see that comment in this page. Can you please investigate why it's not showing up? Or shall I use other channels to file these type of findings?

Thanks!

<img width="1700" alt="Screenshot 2020-03-31 at 15 36 22" src="https://user-images.githubusercontent.com/327285/78032603-919b8100-7365-11ea-8bec-dbb38a6cfcbb.png">

**Reply by becca** [reply]
i invited you to the private feedback repo for maintainers to share feedback - you can use that 🙇 
this is already a known issue but i'll make sure it's prioritized asap.

**Comment by maoo** [comment]
Hi @becca ! QQ. any update on APIs to access discussions data?

**Reply by becca** [reply]
None available just yet but we are working on making this available as soon as possible. We're still in an early beta   period 😊
